internal/test/envtest/environment.go

```diff
@@ -103,11 +103,15 @@ type RunInput struct {
 //	because our tests require access to the *Environment. We use this field to make the created Environment available
 //	to the consumer.
 //
-// Note: Test environment creation can be skipped by setting the environment variable `CAPI_DISABLE_TEST_ENV`. This only
-//
-//	makes sense when executing tests which don't require the test environment, e.g. tests using only the fake client.
+// Note: Test environment creation can be skipped by setting the environment variable `CAPI_DISABLE_TEST_ENV`
+// to a non-empty value. This only makes sense when executing tests which don't require the test environment,
+// e.g. tests using only the fake client.
+// Note: It's possible to write a kubeconfig for the test environment to a file by setting `CAPI_TEST_ENV_KUBECONFIG`.
+// Note: It's possible to skip stopping the test env after the tests have been run by setting `CAPI_TEST_ENV_SKIP_STOP`
+// to a non-empty value.
```

Comment:
can we document those variables also in https://cluster-api.sigs.k8s.io/developer/testing.html#running-unit-and-integration-tests?

Reply:
Done